source/_integrations/advantage_air.markdown

```diff
@@ -49,13 +49,13 @@ The `advantage_air` binary sensor platform will create a binary sensor for each
 
 The `advantage_air` switch platform will create a switch entity to toggle fresh air mode, if it is supported.
 
-## Component services
+## Services
 
-### set_time_to
+### Service `advantage_air.set_time_to`
 
 Set the On/Off Timer using the relevant sensor entity.
 
 | Service data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
-| `entity_id` | no | `sensor.[name]_time_to_on` or `sensor.[name]_time_to_off`
-| `entity_id` | no | Number of minutes between `0` and `720`.
+| `entity_id` | yes | `sensor.[name]_time_to_on` or `sensor.[name]_time_to_off`
```

Comment:
Entity_id is not optional

> Failed to call service advantage_air/set_time_to. must contain at least one of entity_id, area_id.

Reply:
It is optional. It is registered using: `async_register_entity_service`, defined here:

<https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/entity_platform.py#L543>

Which uses:

https://github.com/home-assistant/core/blob/00847db54f00a821f044243696a09638a1847aa2/homeassistant/helpers/config_validation.py#L858-L874

Which has: `vol.Optional(ATTR_ENTITY_ID)`.

Reply:
Ahh yes, I understand now. Thank you.